app/src/main/java/com/example/tasky/itemdetail/presentation/EventDetailScreen.kt

```diff
@@ -0,0 +1,495 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.tasky.R
+import com.example.tasky.agenda.domain.model.Attendee
+import com.example.tasky.itemdetail.domain.VisitorType
+import com.example.tasky.itemdetail.presentation.components.TimeDatePicker
+import com.example.tasky.itemdetail.presentation.components.VisitorRow
+import com.example.tasky.itemdetail.presentation.components.VisitorTypePill
+import com.example.tasky.ui.theme.Black
+import com.example.tasky.ui.theme.DarkGray
+import com.example.tasky.ui.theme.Gray
+import com.example.tasky.ui.theme.Light
+import com.example.tasky.ui.theme.Light2
+import com.example.tasky.ui.theme.LightGray2
+import com.example.tasky.ui.theme.LightGreen
+
+@Composable
+fun EventDetailScreen(
+    state: EventDetailState
+) {
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        containerColor = Black,
+    ) {
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(it),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Row(
+                modifier = Modifier
+                    .padding(
+                        horizontal = 16.dp,
+                        vertical = 12.dp
+                    )
+                    .fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                Icon(
+                    imageVector = Icons.Filled.Close,
+                    tint = Color.White,
+                    contentDescription = stringResource(R.string.close)
+                )
+
+                Text(
+                    text = "01 MARCH 2022",
+                    color = Color.White,
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 16.sp
+                )
+
+                Icon(
+                    imageVector = Icons.Filled.Edit,
+                    tint = Color.White,
+                    contentDescription = stringResource(R.string.edit)
+                )
+            }
+
+            Scaffold(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .clip(
+                        shape = RoundedCornerShape(
+                            topStart = 25.dp,
+                            topEnd = 25.dp
+                        )
+                    ),
+                containerColor = Color.White
+            ) { padding ->
+                LazyColumn(modifier = Modifier.padding(padding)) {
+                    item {
```

Comment:
Random tip:
Everytime you use an "item {}" on a LazyColumn, the stuff inside could be one composable

So in this case you have around 15 item {}, that means you could have around 15 composables (Some are the same stuff so you would reuse them in that case)

For example
```
 Row(
                            modifier = Modifier
                                .height(80.dp)
                                .padding(start = 16.dp),
                            verticalAlignment = Alignment.CenterVertically
                        ) {
                            Box(
                                modifier = Modifier
                                    .size(20.dp)
                                    .clip(RoundedCornerShape(2.dp))
                                    .background(LightGreen)
                            )

                            Spacer(modifier = Modifier.size(10.dp))

                            Text(
                                text = stringResource(id = R.string.event),
                                color = DarkGray,
                                fontWeight = FontWeight.Bold,
                                fontSize = 16.sp
                            )
                        }
```
Could be a composable